spec/jobs/solidus_subscriptions/process_installment_job_spec.rb

```diff
@@ -8,4 +8,22 @@
 
     expect(checkout).to have_received(:process)
   end
+
+  context 'when handling #perform errors' do
+    let(:installment) { build_stubbed(:installment, :failed) }
+
+    before do
+      allow_any_instance_of(described_class).to receive(:perform).and_raise(StandardError.new)
```

Comment:
You're stubbing the method you want to test. That way it will always return `StandardError` instead of running the actual method and rescue it as intended.